### PR TITLE
[Rev 1 Docs] [VnV Report] [VnV Plan] [SRS] [Hazard Analysis] Update tests and requirements for auth; add test results

### LINKED ISSUE #441: [VnV Report] Functional Requirements Evaluation - Authentication
## Content

- Functional Requirements Evaluation

## Rubric Sections

- Content
- Spelling and grammar and style
- VnV Report Matches VnV Plan

### PR DESCRIPTION
**Pull request by LeonSo7**
[Rev 1 Docs] [VnV Report] [VnV Plan] [SRS] Update tests and requirements for auth; add test results

* Add auth functional requirement for logout to SRS
* Remove password reset from SRS
* Add auth test results to VnV report
* Add auth logout FR tests to VnV plan
* Update Hazard Analysis password reset to "system admin resets password" instead of "user resets password"

Close #441

### CODE REVIEW

File: docs/VnVReport/VnVReport.tex [hunk 1]
```diff
@@ -116,6 +117,149 @@ \subsection{Scope of Testing}
 
 \section{Functional Requirements Evaluation}
 
+\subsubsection{Authentication}
+This section will contains tests covering the ``Authentication'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
+
+\begin{enumerate}
+
+	\item \textbf{FRT-BE1-1}
```
Comment (joyxiao99) [inline docs/VnVReport/VnVReport.tex:126]: you forgot pass/fail for all ur tests............

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -512,39 +513,37 @@ \subsubsection{Work Partitioning}
...
+		9                     & Request a quote                & Web Socket     & Database           \\
 		\hline
-		10                    & Request a quote                & Web Socket     & Database           \\
+		10                    & Cancel quote request           & User           & Database           \\
 		\hline
-		11                    & Cancel quote request           & User           & Database           \\
+		11                    & Update quote request           & User           & Database/Websocket \\
 		\hline
-		12                    & Update quote request           & User           & Database/Websocket \\
+		12                    & Copy a quote request           & Database       & Database/Websocket \\
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:532]:
```suggestion
		12                    & Copy a quote request           & Database       & Database           \\
```

File: docs/SRS/SRS.tex [hunk 3]
```diff
@@ -512,39 +513,37 @@ \subsubsection{Work Partitioning}
...
+		6                     & Set appointment availability   & Database       & Database           \\
 		\hline
-		7                     & Set appointment availability   & Database       & Database           \\
+		7                     & View past quotes               & User           & Database           \\
 		\hline
-		8                     & View past quotes               & User           & Database           \\
+		8                     & View quote details             & User           & Database           \\
 		\hline
-		9                     & View quote details             & User           & Database           \\
+		9                     & Request a quote                & Web Socket     & Database           \\
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:526]:
```suggestion
		9                     & Request a quote                & User           & Database           \\
```

File: docs/SRS/SRS.tex [hunk 4]
```diff
@@ -512,39 +513,37 @@ \subsubsection{Work Partitioning}
 		\hline
 		2                     & Login to account               & User           & Database           \\
 		\hline
-		3                     & Reset password                 & User           & Database           \\
+		3                     & Book appointment               & Database       & Database           \\
 		\hline
-		4                     & Book appointment               & Database       & Database           \\
+		4                     & Edit appointment               & Database       & Database           \\
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:516]:
```suggestion
		4                     & Edit appointment               & User           & Database           \\
```

File: docs/SRS/SRS.tex [hunk 5]
```diff
@@ -512,39 +513,37 @@ \subsubsection{Work Partitioning}
...
 		\hline
-		5                     & Edit appointment               & Database       & Database           \\
+		5                     & Cancel appointment             & Database       & Database           \\
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:518]:
```suggestion
		5                     & Cancel appointment             & User           & Database           \\
```

**Comment by LeonSo7** [comment]
Will create an issue to fix work partitioning. Not related to PR changes. I'll also add an item to next meeting to see if we need to change or update the table. @joyxiao99 

https://github.com/arkinmodi/project-sayyara/issues/608

**Review by joyxiao99 — APPROVED** [review]

**Pull request merged**